packages/realm/src/app-services/BaseSubscriptionSet.ts

```diff
@@ -56,19 +56,6 @@ export enum SubscriptionsState {
 
 const DEFAULT_PROPERTY_DESCRIPTOR: PropertyDescriptor = { configurable: true, enumerable: true, writable: false };
 const PROXY_HANDLER: ProxyHandler<BaseSubscriptionSet> = {
```

Comment:
Can we remove the whole Proxy wrapping now?

Reply:
We still need it to enable the spread operator.